src/Microsoft.ML/LearningPipeline.cs

```diff
@@ -109,6 +109,16 @@ internal LearningPipeline(int? seed=null, int conc=0)
         /// <param name="item">Any ML component (data loader, transform or trainer) defined as <see cref="ILearningPipelineItem"/>.</param>
         public void Add(ILearningPipelineItem item) => Items.Add(item);
 
+        /// <summary>
+        /// Add a data loader, transform or trainer into the pipeline.
+        /// </summary>
+        /// <param name="item">Any ML component (data loader, transform or trainer) defined as <see cref="ILearningPipelineItem"/>.</param>
+        /// <returns>Pipeline with added item</returns>
+        public LearningPipeline Append(ILearningPipelineItem item)
+        {
+            Items.Add(item);
```

Comment:
>Items.Add(item); [](start = 12, length = 16)

I'd prefer that instead of duplicating the functionality of `Add` we simply call it. So not `Items.Add` but instead `Add`.